### ISSUE TITLE
#a11y issue - Buttons Not Labeled

### ISSUE THREAD

**Issue by bassettsj** [NEU-Libraries/scholar-onesearch#96]
These items need some screen reader text for the features to be usable by screen readers.

![e-shelf button with voice over not accessible](https://f.cloud.github.com/assets/1218420/1601068/590856ca-5381-11e3-9e82-a22b61cc56f2.PNG)

![e-shelf button with voice over not accessible](https://f.cloud.github.com/assets/1218420/1601069/59090674-5381-11e3-9f5f-501d2277b866.PNG)
